djed-sdk/src/account.js

```diff
@@ -0,0 +1,25 @@
+import { scaledUnscaledPromise, web3Promise } from "./helpers"; 
```

Comment:
Sorry, I didn't notice that I was reviewing the files in "dist", instead of reviewing the files in "src".

Wouldn't it make sense to add the  "dist" folder to ".gitignore", so that it is not pushed to the repo? Or not? I'm not so familiar with the conventions in javascript, but usually automatically generated files and folders are excluded from the repo via ".gitgnore".

Reply:
Yes,so the `dist` folder is created when I build my files using the rollup command.
Including this file will be necessary if we are publishing the SDK to npm(which we will do so !).Hence I've kept the dist folder as it.

Reply:
OK. Perfect!